packages/space-opera/src/components/animation_controls/animation_controls.ts

```diff
@@ -88,39 +84,32 @@ export class AnimationControls extends ConnectedLitElement {
           <me-checkbox id="animation-autoplay" label="Autoplay"
             ?checked="${!!this.config.autoplay}"
             @change=${this.onAutoplayChange}></me-checkbox>
-          <me-checkbox id="animation-play" label="Play animation"
```

Comment:
Removed the play animation toggle, as discussed, to make the model viewer in the editor act as similarly to the snippet as possible.